composer.json

```
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.2",
-        "fakerphp/faker": "^1.13",
+        "fakerphp/faker": "^1.9|^1.13",
```

```suggestion
        "fakerphp/faker": "^1.9",
```
Can you simplify it like that ?